docs/Documentation/Conditions-List.md

```diff
@@ -210,6 +210,15 @@ This condition requires the player to be _below_ specific Y height. The required
     ```YAML
     height 16
     ```
+
+## Hunger: `hunger`
+
+Requires the player to have equal or more hunger points, the condition is the same as `health` just for hunger. If the player is required to have less than the specified amount invert the condition with an exclamation mark (`!`). If the hunger level is below 7, the player cannot sprint.
```

Comment:
I dont like the documentation of the exclamation mark here, because it may change in the future, and it should not be documented in conditions itself, because this could done in every condition